src/components/selector-dropdown/SelectorDropdown.js

```diff
@@ -1,13 +1,15 @@
 // @flow
 import * as React from 'react';
 import classNames from 'classnames';
-import scrollIntoViewIfNeeded from 'scroll-into-view-if-needed';
 import uniqueId from 'lodash/uniqueId';
 
+import { scrollIntoView } from '../../utils/dom';
 import ScrollWrapper from '../scroll-wrapper';
 
 import './SelectorDropdown.scss';
 
+const OVERLAY_WRAPPER_CLASS = 'overlay-wrapper';
```

Comment:
Move this constant to Styles section of src/constants.js, and rename to CLASS_OVERLAY_WRAPPER and then use that